File: app/src/main/java/com/pisces/android/wuha/setting/GradeActivity.java
package com.pisces.android.wuha.setting;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.pisces.android.framworkerlibrary.core.LBaseActivity;
import com.pisces.android.wuha.R;

/**
 * Created by Chris Li on 2017/9/1.
 * 给我评分界面
 */

public class GradeActivity extends LBaseActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_grade);
        setToolbarTitle("给我评分");
    }
}

File: app/src/main/java/com/pisces/android/wuha/setting/GradeActivity.kt
package com.pisces.android.wuha.setting

import android.os.Bundle


import com.pisces.android.wuha.R
import com.pisces.android.wuha.base.LBaseActivity

/**
 * Created by Chris Li on 2017/9/1.
 * 给我评分界面
 */

class GradeActivity : LBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_grade)
        setToolbarTitle("给我评分")
    }
}
